src/ScottLogic.Internal.Training/ScottLogic.Internal.Training.Matcher.Tests/TestMultipleOrders.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
```

Comment:
Do these tests belong in the MatcherTests file? 

This might just be my personal preference but for unit tests I try to only have 1 test file per file. This makes it easy to find the tests for each class. I also tend to structure the test projects to match the actual project, again just to make it easier to find the tests for each file (and it also makes it more obvious what isn't tested)